proprietary/design-tokens/src/imported/product/NLdoc-mapping.json

```diff
@@ -111,22 +111,22 @@
         "level-1": {
           "parent": "viewport (beta)/xs",
           "type": "fontSizes",
-          "value": "2rem"
+          "value": "2.5rem"
         },
         "level-2": {
           "parent": "viewport (beta)/xs",
           "type": "fontSizes",
-          "value": "1.5rem"
+          "value": "2rem"
         },
         "level-3": {
           "parent": "viewport (beta)/xs",
           "type": "fontSizes",
-          "value": "1.25rem"
+          "value": "1.5rem"
         },
         "level-4": {
           "parent": "viewport (beta)/xs",
           "type": "fontSizes",
-          "value": "1rem"
+          "value": "1.25rem"
```

Comment:
```suggestion
          "value": "1rem"
          },
        "level-5": {
          "parent": "viewport (beta)/xs",
          "type": "fontSizes",
          "value": "1rem"
        }
```

Reply:
h4 = 1rem
h5 = 1rem

dan zijn we er denk ik

Reply:
Zoals besproken in Slack voeren we dit niet door. H4 blijft 1.25rem aan de hand van Wendy haar designs